src/components/characters.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./style.css";
+import { Card } from "baseui/card";
+import { cardStyles } from "../styled/style";
+
+type CharacterCardProps = {
+  id: number;
+  image: string;
+  name: string;
+  onClick?: () => void;
```

Comment:
I don't see why `onClick` is marked as optional here. This looks like a mandatory prop that is always passed

Reply:
Mistake noted and corrected